src/Arbor/File/Format/Asif/Type.hs

```diff
@@ -5,7 +5,7 @@ module Arbor.File.Format.Asif.Type where
 import Arbor.File.Format.Asif.Format   (Format)
 import Arbor.File.Format.Asif.Maybe
 import Arbor.File.Format.Asif.Whatever
-import Data.Semigroup
+import Data.Semigroup                  ((<>))
```

Comment:
Most of the time we're using Monoid's version.. is this intentional here?

Reply:
In new versions of GHC, they are the same because `Monoid` is a subclass of `Semigroup.

Reply:
These changes were made to fix warnings because also in new versions of GHC, they've been added to prelude so the import is redundant, but GHC allows for specific imports without throwing a warning.